pkg/controller/statusmanager/pod_status.go

```diff
@@ -144,12 +144,7 @@ func (status *StatusManager) setNodeNetworkUnavailable(nodeName string, ready bo
 	} else {
 		delete(sharedInfo.LastNetworkAvailable, nodeName)
 	}
-	_, changed := status.combineNode(nodeName, ready, reason, message)
-	if !changed {
-		log.Info("Node condition is not changed")
-		return
-	}
-	log.Info(fmt.Sprintf("Setting status NetworkUnavailable to %v for node %s", !ready, nodeName))
+	log.V(1).Info(fmt.Sprintf("Setting status NetworkUnavailable to %v for node %s", !ready, nodeName))
```

Comment:
Please file an issue to update all logs with log level.

Reply:
https://github.com/vmware/nsx-container-plugin-operator/issues/117 was filed